test/StandardToken.js

```diff
@@ -5,8 +5,13 @@ var StandardTokenMock = artifacts.require('./helpers/StandardTokenMock.sol');
 
 contract('StandardToken', function(accounts) {
 
+  let token;
```

Comment:
I think that's OK. 100 wei is a very small value. why do you think it could run out of funds?